knowledge-base/editor-missing-br-in-value.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Missing BR Tags in Editor HTML Value
+description: Missing BR Tags in Editor HTML Value
+type: troubleshooting
+page_title: Missing BR Tags in Editor HTML Value
+slug: editor-kb-missing-br-tags-in-value
+position: 
+tags: telerik, blazor, editor
+ticketid: 1628534
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>Editor for Blazor</td>
+        </tr>
+        <tr>
+            <td>Product Version</td>
+            <td>4.6.0 +</td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+This KB article addresses the following questions or issues:
+
+* Multiple newlines in the Editor are represented by empty paragraphs and are not visible when rendered outside the Editor.
+* Why empty paragraphs look different in the Editor and outside the Editor?
+* Why empty paragraphs in the Editor don't show afterwards on the web page?
+
+
+## Cause/Possible Causes
+
+By default, empty paragraphs do not occupy any space on the web page. Here is why empty paragraphs look different inside and outside the Editor:
+
+Note that the Editor uses the [ProseMirror engine](https://prosemirror.net). By design, the engine inserts a `<br />` tag in every empty `<p></p>`. This ensures that each empty paragraph takes up visible space and the user can distinguish it, focus it and type inside. Since version 4.6.0, the Editor `Value` does not contain these "service" `<br />` tags anymore.
```

Comment:
```suggestion
Note that the Editor uses the [ProseMirror engine](https://prosemirror.net). By design, the engine inserts a `<br />` tag in every empty `<p></p>`. This ensures that each empty paragraph takes up visible space and the user can distinguish it, focus it, and type inside. Since version 4.6.0, the Editor `Value` does not contain these "service" `<br />` tags anymore.
```